Sources/MapboxMaps/Annotations/AnnotationOrchestrator.swift

```diff
@@ -39,18 +37,6 @@ internal protocol AnnotationManagerInternal: AnnotationManager {
     func handleDragEnded()
 }
 
-/// A delegate that is called when a tap is detected on an annotation (or on several of them).
-public protocol AnnotationInteractionDelegate: AnyObject {
```

Comment:
Removing this public protocol would be a breaking change. IMO this is still useful - this protocol is useful when one is interested in reacting to tap events.

Reply:
Gotcha, fixed!